sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -0,0 +1,1299 @@
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "string";
+                        readonly description: "Alternative name of merchant";
+                        readonly example: "Contoso (R)";
+                    };
+                };
+                readonly Items: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "object";
+                        readonly description: "Extracted line item";
+                        readonly example: "1\nSurface Pro 6\n$999.00\n$999.00";
+                        readonly properties: {
+                            readonly TotalPrice: {
+                                readonly type: "number";
+                                readonly description: "Total price of line item";
+                                readonly example: "$999.00";
+                            };
+                            readonly Name: {
+                                readonly type: "string";
+                                readonly description: "Item name";
+                                readonly example: "Surface Pro 6";
+                            };
+                            readonly Quantity: {
+                                readonly type: "number";
+                                readonly description: "Quantity of each item";
+                                readonly example: "1";
+                            };
+                            readonly Price: {
+                                readonly type: "number";
+                                readonly description: "Individual price of each item unit";
+                                readonly example: "$999.00";
+                            };
+                            readonly Description: {
+                                readonly type: "string";
+                                readonly description: "Item description";
+                                readonly example: "Room Charge";
+                            };
+                            readonly Date: {
+                                readonly type: "date";
+                                readonly description: "Item date";
+                                readonly example: "27Mar21";
+                            };
+                            readonly Category: {
+                                readonly type: "string";
+                                readonly enum: readonly ["Room", "Meals", "Tax", "Deposit", "Credit", "Other"];
+                                readonly description: "Item category";
+                                readonly example: "Room";
+                            };
+                        };
+                    };
+                };
+            };
+        };
+    };
+};
+
+// @public
+export type ReifyFieldSchema<Schema extends Readonly<FieldSchema>> = Schema extends StructuredStringFieldSchema<infer Type> ? {
```

Comment:
Is this type expected to be used by our customers? if not, let's tag it with `@hidden`.

Reply:
It appears in some signatures, notably the type signature of Receipt, Invoice, BusinessCard, and IdentityDocument, but I can mark it hidden.

Reply:
Two reasons for my suggestion:
- do you expect your customers to use that type function directly? I hope not so I believe it is ok for them to not see it
- I am worried about how it would look in the docs website.

Reply:
Well, my hope is that by beta 2 or 3, we can actually just remove this entirely in favor of ahead-of-time model generation from the service's schemas that doesn't expose such a complicated type. I've marked it hidden now. I'm not sure what will happen though if someone goes looking for this type and cant find it, though.